src/Serialization.js

```diff
@@ -297,5 +289,72 @@ exports.newMoveInstantaneousRewardsCertificate = mir => () =>
 exports.newWithdrawals = containerHelper => entries => () =>
     containerHelper.packMap(lib.Withdrawals, entries);
 
-exports.setTxBodyWithdrawals = txBody => withdrawals => () =>
-    txBody.set_withdrawals(withdrawals);
+exports.setTxBodyWithdrawals = setter('withdrawals');
+
+exports.setTxBodyUpdate = setter('update');
+
+exports.newUpdate = ppUpdates => epoch => () =>
+    lib.Update.new(ppUpdates, epoch);
+
+exports.ppu_set_minfee_a = setter('minfee_a');
```

Comment:
This applies to a significant portion of this module and the corresponding FFI imports: I'd prefer not to use snake_case anywhere